src/app/components/timeline/BeatsCell.js

```diff
@@ -24,39 +24,39 @@ const {
   isMediumSelector,
 } = selectors
 
-class ScenesCell extends PureComponent {
+class BeatsCell extends PureComponent {
```

Comment:
This one actually should stay ScenesCell (or something like that - CardsCell?) because it's referring to a scene card

Reply:
I changed `BeatCardAdd` and `BeatsCell` to `CardAdd` and `CardCell` respectively.
I like the consistency that this has to `cards` in the store.
If we used `scene`, the reader might be misled into believing that another concept is encoded in the name.